webui/react/src/components/ColumnPickerMenu.tsx

```diff
@@ -95,18 +96,44 @@ const ColumnPickerTab: React.FC<ColumnTabProps> = ({
   }, [searchString, totalColumns, tab]);
 
   const allFilteredColumnsChecked = useMemo(() => {
-    return filteredColumns.every((col) => columnState.includes(col.column));
+    return filteredColumns.every((col) => {
+      const colType = col.type.replace('COLUMN_TYPE_', '');
+
+      if (col.column.includes('metadata'))
+        return columnState.includes(col.column.concat(`_${colType}`));
```

Comment:
please read the epic ticket. we need to be careful about how we encode the column name because the backend does not place any limitations about the column name, so if the user includes a type name in the column name, this could become problematic.

use the method discussed in the ticket to encode the type/column name tuple.